src/plugin/ipc/socket/socketconnection.cpp

```diff
@@ -418,6 +418,57 @@ TcpConnection::TcpConnection(const TcpConnection &parent,
 }
 
 void
+TcpConnection::sendPeerInformation()
+{
+  struct sockaddr local, remote;
+  socklen_t localsz, remotesz;
+  switch (_type) {
+  case TCP_CONNECT:
+  {
+    localsz = sizeof(local);
+    JASSERT(getsockname(_fds[0],
```

Comment:
Is value returned by `getsockname` name guaranteed to match with the value returned by `getpeername`?